src/components/cc-oauth-consumer/cc-oauth-consumer.stories.js

```diff
@@ -0,0 +1,96 @@
+import { makeStory } from '../../stories/lib/make-story.js';
```

Comment:
note: for the whole file, in many files we order stories like this:

* default
* loading
* error
* dataLoaded